### PR TITLE
CLI: Improved error control and updated hub requirement

### PR DESCRIPTION
**Pull request by gante**
# What does this PR do?

This PR adds three things to the `pt-to-tf` CLI:
- Updates the hub minimum version requirement, so that all users can adequately use this CLI (the new `0.9.0` is a requirement to open a PR with a non-god token)
- Adds more flexible admissible error control (some models were failing with errors like 6e-5, which was annoying. The `5e-5` is a magic number anyways)
- Improves the commit message, in case the user fiddles with the admissible error

### CODE REVIEW

File: src/transformers/commands/pt_to_tf.py [hunk 1]
```diff
@@ -96,12 +96,11 @@ def register_subcommand(parser: ArgumentParser):
...
         )
         train_parser.add_argument(
-            "--max-hidden-error",
+            "--max-error",
             type=float,
             default=MAX_ERROR,
             help=(
-                f"Maximum error tolerance for hidden layer outputs. Defaults to {MAX_ERROR}. If you suspect the hidden"
-                " layers outputs will be used for downstream applications, avoid increasing this tolerance."
+                f"Maximum error tolerance. Defaults to {MAX_ERROR}. This flag should be avoided, use at your own risk."
```
Comment (amyeroberts) [inline src/transformers/commands/pt_to_tf.py:103]: ☠️

File: src/transformers/commands/pt_to_tf.py [hunk 2]
```diff
@@ -348,6 +348,10 @@ def run(self):
                 f"Maximum conversion output difference={max_conversion_output_diff:.3e}; "
                 f"Maximum conversion hidden layer difference={max_conversion_hidden_diff:.3e};\n"
             )
+            if self._max_error > MAX_ERROR:
+                commit_descrition += (
```
Comment (amyeroberts) [inline src/transformers/commands/pt_to_tf.py:352]: Nice 👍  :)

**Review by amyeroberts — APPROVED** [review]
LGTM 👍

**Review by ydshieh — APPROVED** [review]
Great, @gante !

**Pull request merged**